spree_amazon_payments.gemspec

```diff
@@ -14,7 +14,9 @@ Gem::Specification.new do |s|
   s.version     = '2.3.4'
```

Comment:
So if I wanted to merge this right into master, shouldnt we rename our version to 3.0.0?